src/components/frame/v5/pages/StreamingPaymentsPage/StreamingPaymentsPage.tsx

```diff
@@ -29,16 +33,30 @@ const StreamingPaymentsPage = () => {
     nativeDomainId,
   });
 
+  const { extensionData } = useExtensionData(Extension.StreamingPayments);
+  const isStreamingPaymentsInstalled =
+    extensionData && isInstalledExtensionData(extensionData);
```

Comment:
I believe this piece of code can be easily refactored to use 

```
const { isExtensionInstalled } = useExtensionItem(
    Extension.StreamingPayments,
  );
```

Given it performs already all these checks to verify if the extension was or not installed.